gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -88,6 +88,10 @@ public final File getOutputFile() {
     @Optional
     public abstract MapProperty<String, Serializable> getConjureExtensions();
 
+    @Input
+    @Optional
+    public abstract MapProperty<String, Object> getOptions();
```

Comment:
In practice I suspect the places using `GeneratorOptions` as an input would benefit from `MapProperty<String, Object>`, but I didn't update existing consumers in this PR.